webhookbot/webhookbot/http.go

```diff
@@ -83,7 +80,39 @@ func (h *HTTPSrv) handleHook(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 	h.Stats.Count("handle - success")
-	h.ChatEcho(hook.convID, "[hook: *%s*]\n\n%s", hook.name, msg)
+	if _, err := h.Config().KBC.SendMessageByConvID(hook.convID, "[hook: *%s*]\n\n%s", hook.name, msg); err != nil {
+		// error created in https://github.com/keybase/client/blob/1985b18c4e7659bede1d4a2e68e4f68467acebc6/go/client/chat_svc_handler.go#L1407
+		// error created in https://github.com/keybase/keybase/blob/9a82c96231ea2c6132532002e58bac80849265e6/go/chatbase/storage/sql_chat.go#L2324
+		if strings.Contains(err.Error(), "no conversations matched") ||
```

Comment:
this should be in helpers/happen at a higher level, doesn't githubbot already do this?

Reply:
It happens in ChatEcho, but that doesn't return an error. I was thinking that sending a message as an attachment if it's too long should be specific to webhookbot, not to all bots.

Reply:
We can just move this kind of error checking for no conversation match to a helper that ChatEcho and this function use.